static/js/containers/FinancialAidCalculator.js

```diff
@@ -109,6 +124,16 @@ const validationMessage = (key, errors) => {
   </div>;
 };
 
+const apiError = ({ message, code }: FetchError) => (
+  <div className="api-error">
```

Comment:
haha, definitely out of scope. We don't use `<form>` tags anywhere on the site, except perhaps on the homepage for the mailchimp signup dialog